R/crs.R

```diff
@@ -164,7 +164,9 @@ st_crs.default = function(x, ...) NA_crs_
 make_crs = function(x) {
 
 	if (inherits(x, "CRS")) {
-		x = if (!is.null(comment(x)))
+		x = if (is.null(comment(x)) || (CPL_proj_version() < "6.0.0" || 
```

Comment:
If `comment(x)` is NULL, you assign NULL to x, which breaks on line 177.

Reply:
OK, I'll reverse them.